core/systems/frzr/frzr.gd

```diff
@@ -35,6 +35,8 @@ func get_available_disks() -> Array[Disk]:
 		if not line.contains("disk"):
 			continue
 		var parts := line.split(" ", false, 3)
+		if parts[2] as int == 0:
```

Comment:
The output of `lsblk --list -n -o name,type,size,model` will show the size along with a unit (e.g. 1G, 0B). This should be updated to match on that pattern, instead of casting to an int.

E.g.
```suggestion
		if parts[2] == "0B":
```

Reply:
Done! Thanks!

I observed that in GDscript if you pass it a string as int it will remove all non-int characters, is my observation correct? I am unable to replicate the behaviour in Python.